docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -1619,7 +1619,34 @@ To enable Web Links support, add the `quarkus-resteasy-reactive-links` extension
 
 |===
 
-Importing this module will allow injecting web links into the response HTTP headers by just annotating your endpoint resources with the `@InjectRestLinks` annotation. To declare the web links that will be returned, you need to use the `@RestLink` annotation in the linked methods. An example of this could look like:
+Importing this module will allow injecting web links into the response HTTP headers by just annotating your endpoint resources with the `@InjectRestLinks` annotation. To declare the web links that will be returned, you must use the `@RestLink` annotation in the linked methods.
+Assuming a `Record` looks like:
+
+[source,java]
+----
+public class Record {
+
+    // id is a mandatory field for the generation of web links
```

Comment:
Looking at the implementation, this does not appear to be true. ATM it's limited in that the endpoint `@PathParam` parameter must be named `id`, but the field from the entity has to be annotated with `@Id`, if I understand it correctly. This doesn't appear to be documented, though, so further testing should be done to validate this.

Reply:
The original issue comes with a project to reproduce the problem and there the endpoint that throws the exception is defined with an `@PathParam("id")` already.
I tested adding the @jakarta.persistence.Id to the field in the `TestRecord` class (called `name`) and the endpoint works properly that way.
So what seem to be needed for the web links to work is either a field called `id` or an @Id annotation (that ends with `persistence.Id`)